helm/reana/templates/ui-config.yaml

```
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ui-config
+  namespace: {{ .Release.Namespace }}
+data:
+  ui-config.yaml: |
+    announcement: {{ .Values.components.reana_ui.announcement | quote }}
```

Now this one (and the rest endint just quoting) have no default, you can do the following:
```suggestion
    announcement: {{ .Values.components.reana_ui.announcement | quote | default "null" }}
```